tests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests/NHSD.BuyingCatalogue.Ordering.Api.IntegrationTests.csproj

```diff
@@ -19,9 +23,15 @@
   </ItemGroup>
 
   <ItemGroup>
+    <EmbeddedResource Include="public_privatekey.pfx" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <PackageReference Include="AWSSDK.Core" Version="3.3.106.22" />
```

Comment:
Are we switching to AWS? 😄 

In all seriousness, do we need this package?

Reply:
I just love AWS!

… will remove it

Reply:
and I always had you as a Microsoft guy.... I guess looks can be deceiving 😄